src/EventHub.Application/Events/EventAppService.cs

```diff
@@ -277,5 +401,60 @@ private async Task SaveCoverImageAsync(Guid id, IRemoteStreamContent streamConte
 
             await _eventBlobContainer.SaveAsync(blobName, streamContent.GetStream(), overrideExisting: true);
         }
+        
+        private async Task<List<Guid>> GetUserIdsByUserNamesAsync(List<string> userNames)
+        {
+            var userQueryable = await _userRepository.GetQueryableAsync();
+
+            var query = userQueryable
+                    .Where(u => userNames.Any(p => p == u.UserName))
```

Comment:
I think Contains is better than Any. However, that code doesn't guarantee that the returned List of Guids has same element count with the List of strings (usernames). What if a username doesn't exists in the database?